lib/src/analytics.dart

```diff
@@ -281,13 +297,15 @@ class TestAnalytics extends AnalyticsImpl {
     // Calling the [generateRequestBody] method will ensure that the
     // session file is getting updated without actually making any
     // POST requests to Google Analytics
-    generateRequestBody(
+    final Map<String, dynamic> body = generateRequestBody(
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic

```suggestion
    final Map<String, Object?> body = generateRequestBody(
```

Reply:
And throughout this file, please :)